### ISSUE TITLE
Move Build Process To Python3

### ISSUE THREAD

**Issue by MorganRodgers** [OSC/ood-documentation#91]
Python 2.x is scheduled for end of life in 2020 (meaning no additional bug fixes), and so we should remove references to Python 2.x from the build process.

https://legacy.python.org/dev/peps/pep-0373/

**Comment by ericfranz** [comment]
Works for me. We have rh-python35 on apps-test.awesim.org and can easily install . It would be nice to have directions that work from this host too.

```console
$ yum list rh-python35\*sphinx\*

There is no pipenv rpm but there are these:

...
Available Packages
rh-python35-python-sphinx.noarch                                                                               1.2.2-2.el6                                                                            rhel-server-rhscl-6-rpms
rh-python35-python-sphinx-doc.noarch                                                                           1.2.2-2.el6                                                                            rhel-server-rhscl-6-rpms
```

**Comment by ericfranz** [comment]
Though getting the build to work easily on apps-test.awesim.org vs just updating to use python 3 are two separate tasks so if it is difficult I'd open a separate issue for that.

**Comment by MorganRodgers** [comment]
Implemented in #163 and https://github.com/OSC/docker-sphinx/pull/1.
